File: snuggle_server/snuggle_server/src/main/java/com/ssafy/snuggle/model/service/TaggingService.java
package com.ssafy.snuggle.model.service;

import com.ssafy.snuggle.model.dto.Tagging;

public interface TaggingService {

	 /**
     * NFC 태깅된 영상에 대한 정보를 단건 조회한다.
     * @param taggingId
     * @return
     */
    Tagging selectByTaggingId(String taggingId);
    
}

File: snuggle_final_app/app/src/main/java/com/ssafy/snuggle_final_app/data/service/TaggingService.kt
package com.ssafy.snuggle_final_app.data.service

import com.ssafy.snuggle_final_app.data.model.dto.Tagging
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface TaggingService {
    @GET("snuggle/tagging/{tagId}")
    suspend fun getTaggingById(@Path("tagId") taggingId: String): Tagging
}